libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.spec.ts

```diff
@@ -10,6 +10,7 @@ import {
 import { By } from '@angular/platform-browser';
 import { expect } from '@skyux-sdk/testing';
 import { SkyMediaBreakpoints, SkyMediaQueryService } from '@skyux/core';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
```

Comment:
Why does this need to be added here?

Reply:
This was just for a warning that the IDE was throwing. It appeared to be because the testing library it is coming from isn’t a true library and the tests are in one. However, the tests aren’t actually built so it is ok.